src/Event.php

```diff
@@ -510,7 +511,11 @@ private static function getTypeValuesForDropdown(): array
         $itemtypes = [];
         foreach (self::getUsedItemtypes() as $value) {
             $itemtype = self::getItemtypeFromType($value);
-            $itemtypes[$value] = $itemtype::getTypeName(1);
+            if ($itemtype) {
+                $itemtypes[$value] = $itemtype::getTypeName(1);
+            } else {
+                trigger_error("Unsupported type: $value", E_USER_WARNING);
```

Comment:
Maybe entry should be added using `$itemtypes[$value] = $value`, to still be able to filter on this value.